docker/precommit_hooks/Dockerfile

```diff
@@ -42,6 +43,8 @@ WORKDIR /code
 ENV PRE_COMMIT_COLOR=always
 ENV SKIP=test-download-from-gcs-helper,check-bug-in-commit-message,run-py2-tests,check-if-starboard-interface-changed
 
+RUN git config --global --add safe.directory /code
```

Comment:
is this needed in the repo or you needed it locally ?

Reply:
This is needed in any container that runs git commands on the checked out code. We had to put this in in every other container - see docsite and others.